package.json

```diff
@@ -65,7 +65,7 @@
     "mpd-parser": "0.6.1",
     "mux.js": "4.4.1",
     "url-toolkit": "^2.1.3",
-    "video.js": "^6.8.0 || ^7.0.0"
+    "video.js": "^6.8.0 || ^7.0.3"
```

Comment:
Just need to update this to 7.0.4 (or whatever version the changes in `video.core.js` are in) when the time comes.

Reply:
Well this really indicates what versions of video.js are compatible with VHS, I should actually bump this down to `^7.0.0` and update the lock file to 7.0.4 when that's out